packages/twenty-server/src/engine/core-modules/billing/stripe/stripe.service.ts

```diff
@@ -114,7 +114,10 @@ export class StripeService {
       success_url: successUrl,
       cancel_url: cancelUrl,
     });
-  }
+  } // I prefered to not create a customer with metadat before the checkout, because it would break the tax calculation
```

Comment:
understand the comment but why is it in here?

Reply:
because Felix suggested this method for adding the customer metadata